cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java

```diff
@@ -0,0 +1,281 @@
+        + "Ballerina Central")
+public class PushCommand implements BLauncherCmd {
+    private static PrintStream outStream = System.err;
+
+    @CommandLine.Parameters
+    private List<String> argList;
+
+    @CommandLine.Option(names = {"--help", "-h"}, hidden = true)
+    private boolean helpFlag;
+
+    @CommandLine.Option(names = "--debug", hidden = true)
+    private String debugPort;
+
+    @CommandLine.Option(names = "--repository", hidden = true)
+    private String repositoryHome;
+
+    @CommandLine.Option(names = {"--skip-source-check"}, description = "skip checking if source has changed")
+    private boolean skipSourceCheck;
+
+    @CommandLine.Option(names = "--experimental", description = "enable experimental language features")
+    private boolean experimentalFlag;
+
+    private Path userDir;
+    private PrintStream errStream;
+    
+    public PushCommand() {
+        userDir = Paths.get(System.getProperty("user.dir"));
+        errStream = System.err;
+    }
+    
+    public PushCommand(Path userDir, PrintStream errStream) {
+        this.userDir = userDir;
+        this.errStream = errStream;
+    }
+
+    @Override
+    public void execute() {
+        if (helpFlag) {
+            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(PUSH_COMMAND);
+            outStream.println(commandUsageInfo);
+            return;
+        }
+
+        // Get source root path.
+        Path sourceRootPath = userDir;
+
+        // Push command only works inside a project
+        if (!ProjectUtils.isBallerinaProject(sourceRootPath)) {
+            Path findRoot = ProjectUtils.findProjectRoot(sourceRootPath);
+            if (null == findRoot) {
+                CommandUtil.printError(errStream,
+                        "Push command can be only run inside a Ballerina project",
+                        null,
+                        false);
+                return;
+            }
+            sourceRootPath = findRoot;
+        }
+
+        // todo: load project and get src path and all other stuff
+        // todo: get target path and balo path using project
+        BuildProject project = BuildProject.loadProject(sourceRootPath);
+
+        // Enable remote debugging
+        if (null != debugPort) {
+            System.setProperty(SYSTEM_PROP_BAL_DEBUG, debugPort);
+        }
+
+        if (argList == null || argList.isEmpty()) {
+            pushPackage(project);
+        } else {
+            throw LauncherUtils.createUsageExceptionWithHelp("too many arguments");
+        }
+
+        // Exit status, zero for OK, non-zero for error
+        Runtime.getRuntime().exit(0);
+    }
+
+    @Override
+    public String getName() {
+        return PUSH_COMMAND;
+    }
+
+    @Override
+    public void printLongDesc(StringBuilder out) {
+        out.append("push modules to Ballerina Central");
+    }
+
+    @Override
+    public void printUsage(StringBuilder out) {
+        out.append("  ballerina push <module-name> \n");
```

Comment:
There is no argument to pass right ? Should it be just `ballerina push` ?

Reply:
removed `<module-name>`